test/distributed/_shard/sharded_tensor/ops/test_linear.py

```diff
@@ -65,29 +67,24 @@ def _run_sharded_linear(
         # Shard the parameter.
         shard_parameter(sharded_linear, "weight", spec)
 
+        # Wrap with DDP enabled with ReplicatedTensor
+        with _ddp_replicated_tensor(True):
+            sharded_linear = DDP(sharded_linear)
```

Comment:
Does this DDP will automatically wrap `bias` as a Replicated Tensor? What if users don't use DDP, does that mean we don't even allow users to use `bias` as a `torch.Tensor`?

Reply:
It doesn't do it automatically, you need `with _ddp_replicated_tensor(True):`. The tricky thing is that if users don't use DDP, we kinda end up assuming `bias` is still replicated.

I do feel it makes things cleaner if users always use DDP and we recommend users to always combine DDP with sharding and not use sharding alone, because then it results in things like us treating `bias` implicitly as a ReplicatedTensor and I feel its better to have things be explicit.